backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/validation/LanguageCodeValidator.java

```diff
@@ -32,7 +33,7 @@ public void initialize(LanguageCode constraintAnnotation) {
 
 	@Override
 	public boolean isValid(String value, ConstraintValidatorContext context) {
-		if (value == null) { return true; }
+		if (value == null || StringUtils.isBlank(value)) { return true; }
```

Comment:
Since `StringUtils.isBlank(..)` handles the `null` check:

```suggestion
		if (StringUtils.isBlank(value)) { return true; }
```

Reply:
done